templates/settings-admin.php

```diff
@@ -26,54 +26,189 @@
+	</p><p>
+		<?php p($l->t('CPU').': '); ?>
+		<span class="info"><?php p($_['cpu']); ?></span>
+	</p><p>
+		<?php p($l->t('Memory').': '); ?>
+		<span class="info"><?php p($_['memory']); ?></span>
+	</p><p>
+		<?php p($l->t('Server time').': '); ?>
+		<span class="info" id="servertime"></span>
+	</p><p>
+		<?php p($l->t('Uptime').': '); ?>
+		<span class="info" id="uptime"></span>
+	</p><p>
+		<?php p($l->t('Time Servers').': '); ?>
+		<span class="info" id="timeservers"></span>
+	</p>
+
+
+
+	<div class="wrapper">
+	    <div id="one">
+
+			<div class="infobox" id="cpuSection">
+				<h2><?php p($l->t('Load'));?></h2>
+				<canvas id="cpuloadcanvas" width="100" height="150"></canvas>
+			</div>
+			<p><em id="cpuFooterInfo"></em></p>
+
+		</div>
+		<div id="two">
+
+			<div class="infobox">
+			<h2><?php p($l->t('Memory'));?></h2>
+			<canvas id="memorycanvas" width="300" height="150"></canvas>
+			</div>
+			<p><span class="rambox" id="rambox">&nbsp;&nbsp;</span>&nbsp;&nbsp;<em id="memFooterInfo"></em></p>
+			<p><span class="swapbox" id="swapbox">&nbsp;&nbsp;</span>&nbsp;&nbsp;<em id="swapFooterInfo"></em></p>
+
+
+		</div>
+	</div>
+
+
+
 </div>
-<div class="section" id="memorySection">
-	<h2><?php p($l->t('Memory usage'));?></h2>
-	<canvas id="memorycanvas" width="600" height="150"></canvas>
-	<p><img src="data:image/png;base64,iVBORw0KGgoAAAANSUhEUgAAAAoAAAAKCAIAAAACUFjqAAAACXBIWXMAAAsTAAALEwEAmpwYAAAAB3RJTUUH4QYBDCANxB9ROQAAABl0RVh0Q29tbWVudABDcmVhdGVkIHdpdGggR0lNUFeBDhcAAAAUSURBVBjTY2T4z4AHMDEwjEpjAgBALQETBkU7DgAAAABJRU5ErkJggg==" alt="RAM color" />&nbsp;&nbsp;<em id="memFooterInfo"></em></p>
-	<p><img src="data:image/png;base64,iVBORw0KGgoAAAANSUhEUgAAAAoAAAAKCAIAAAACUFjqAAAACXBIWXMAAAsTAAALEwEAmpwYAAAAB3RJTUUH4QYBDB0ds0AJ4wAAABl0RVh0Q29tbWVudABDcmVhdGVkIHdpdGggR0lNUFeBDhcAAAASSURBVBjTY/zPgA8wMYxKYwEAQSwBE2ViEDMAAAAASUVORK5CYII=" alt="SWAP color" />&nbsp;&nbsp;<em id="swapFooterInfo"></em></p>
+
+<div class="section"><h2><?php echo('<img class="infoicon" src="'.image_path('serverinfo', 'hdd-o.svg').'">'); p($l->t('Disk')); ?></h2>
+	<p>
+		<?php
+		foreach ($_['diskinfo'] as $disk) {
+
+			echo('<div class="infobox">');
+
+			echo('<div class="diskchart-container">');
+			echo('<canvas id="DiskChart" class="DiskChart" width="50" height="50"></canvas>');
+			echo('</div>');
+
+			echo('<h3>'.basename($disk['device']).'</h3>');
+			echo('<p>');
+			p($l->t('Mount').': ');
+			echo('<span class="info">'.$disk['mount'].'</span>');
+			echo('</p><p>');
+			p($l->t('Filesystem').': ');
+			echo('<span class="info">'.$disk['fs'].'</span>');
+			echo('</p><p>');
+			p($l->t('Size').': ');
+			echo('<span class="info">'.FormatBytes($disk['used']+$disk['available']).'</span>');
+			echo('</p><p>');
+			p($l->t('Available').': ');
+			echo('<span class="info">'.FormatBytes($disk['available']).'</span>');
+			echo('</p><p>');
+			p($l->t('Used').': ');
+			echo('<span class="info">'.$disk['percent'].'</span>');
+			echo('</p></div>');
```

Comment:
It's maybe better to read to leave PHP and write plain HTML instead of the `echo`'s on every line.